src/lib.rs

```diff
@@ -60,49 +60,44 @@
 //! ```
 
 extern crate cgmath;
+extern crate gltf_json as json;
 #[macro_use]
 extern crate lazy_static;
 
-/// Contains (de)serializable data structures that match the glTF JSON text.
-pub extern crate gltf_json as json;
```

Comment:
Technically a breaking change. Should this be 0.10.0 instead?

Reply:
Actually, I'll put this back as I'd rather have a 0.9.1 version than a 0.10 version. Let's deprecate instead.